docs/docs/dev/deployments/multi-zone.md

```diff
@@ -1,13 +1,127 @@
-# Set up a multi-zone deployment
+# Multi-zone deployment
 
-For a description of how multi-zone deployments work in Kuma, see [about multi-zone deployments](../how-multi-zone-works). This page explains how to configure and deploy Kuma in a multi-zone environment:
+## About
+
+Kuma supports running your service mesh in multiple zones. It is even possible to run with a mix of Kubernetes and Universal zones. Your mesh environment can include multiple isolated service meshes (multi-tenancy), and workloads running in different regions, on different clouds, or in different datacenters. A zone can be a Kubernetes cluster, a VPC, or any other deployment you need to include in the same distributed mesh environment.
+
+<center>
+<img src="/images/docs/0.6.0/distributed-diagram.jpg" alt="" style="width: 500px; padding-top: 20px; padding-bottom: 10px;"/>
+</center>
+
+### How it works
+
+Kuma manages service connectivity -- establishing and maintaining connections across zones in the mesh -- with the zone ingress and with a DNS resolver.
+
+The DNS resolver is embedded in each data-plane proxy and configured through XDS. It resolves each service address to a virtual IP address for all service-to-service communication.
+
+The global control plane and the zone control planes communicate to synchronize resources such as Kuma policy configurations over Kuma Discovery Service (KDS), which is a protocol based on xDS.
+
+:::tip
+A zone ingress is not an API gateway. Instead, it is specific to internal cross-zone communication within the mesh. API gateways are supported in Kuma [gateway mode](../documentation/dps-and-data-model.md) which can be deployed in addition to zone ingresses.
+:::
+
+### Components of a multi-zone deployment
+
+A multi-zone deployment includes:
+
+* The **global control plane**:
+  * Accept connections only from zone control planes.
+  * Accept creation and changes to [policies](/policies) that will be applied to the data plane proxies.
+  * Send policies down to remote control-planes.
```

Comment:
It should be `zone control-planes`. There are a couple of occurrences in the file.